src/app/game/simon/page.tsx

```
@@ -0,0 +1,441 @@
+      </div>
+
+      {gameState === 'idle' && (
+        <div className={styles.menu}>
+          <div className={styles.difficultySelector}>
+            <h3>Select Difficulty</h3>
+            {Object.entries(DIFFICULTY_CONFIG).map(([key, config]) => (
+              <button
+                key={key}
+                className={`${styles.difficultyButton} ${difficulty === key ? styles.selected : ''}`}
+                onClick={() => setDifficulty(key as Difficulty)}
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)